src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -393,6 +393,17 @@ public DebuggedProcess(bool bLaunched, LaunchOptions launchOptions, ISampleEngin
                 {
                     message = result.Results.TryFindString("message");
                 }
+                // if the command was abort (usually because of breakpoints failing to bind) then gdb writes messages into the output
+                if (this.MICommandFactory.Mode == MIMode.Gdb && message == "Command aborted.")
+                {
+                    message = MICoreResources.Error_CommandAborted;
+                    if (ProcessState == ProcessState.Running)
```

Comment:
Should this be ProcessState == stopped or should we add an exception handling to the response for continue instead of doing it here? I think we run the risk of getting any command abort in this state that can indicate something else is wrong with gdb. Maybe add another check to see if we know we are in this specific scenario?

Reply:
== Running is correct. The problem is that the "^error" arrives AFTER the command has completed with a "^running" result (see log in description above), so we can't handle this as a response to the continue command itself. We could have some kind of "last command" variable and check for a continue op, but that is a pretty ugly solution too.  I have only ever seen this behavior happen in continue/step operations. Also I have looked at the gdb sources and only see this string generated when stepping or continuing. So I think it is pretty safe to make this assumption. But happy to consider alternatives.